desktop/apps/marketing_signup_modals/test/fixture.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
Might be okay to leave this out considering we're not requiring it (although it is a nice reference to look at).